MoviesProject/Home/HomeView.swift

```diff
@@ -19,67 +17,56 @@ public extension Home {
                     case .idle:
                         Text("idle")
                     case .loading:
-                        ProgressView()
-                            .foregroundColor(.white)
-                        
+                        Color.clear
+                            .overlay {
+                                ProgressView()
+                            }
+                            .background(.darkBackground)
                     case .loaded(let movies):
-                        
-//                        MovieDetails.MovieDetailsView(store: store.scope(state: \.movieDetails, action: Action.movieDetails))
-                        
-                        Text("loaded")
-                        NavigationView {
-                            ScrollView {
-                                VStack() {
-                                    //                    TextField("Search", text: $searchText)
+                        ScrollView {
+                            VStack() {
+                                Button(action: {
+                                    viewStore.send(.searchBarTapped(movies))
+                                    print("Button tapped")
+                                }) {
                                     HStack(){
-                                        
-                                        Button("Search") {
-                                            viewStore.send(.searchBarTapped(movies[0]))
-//                                            MovieDetails.MovieDetailsView(store: store.scope(state: \.movieDetails, action: Action.movieDetails))
-                                        }
-                                        .padding(EdgeInsets(top: 12, leading: 24, bottom: 12, trailing: 0))
-                                        .foregroundColor(.lightGray)
-                                        .font(.custom("Poppins", fixedSize: 14))
+                                        Text("Search")
+                                            .padding(EdgeInsets(top: 12, leading: 24, bottom: 12, trailing: 0))
+                                            .foregroundColor(.lightGray)
+                                            .font(.custom("Poppins", fixedSize: 14))
                                         
                                         Image(.searchIcon)
                                             .frame(maxWidth: .infinity, alignment: .trailing)
                                             .padding(EdgeInsets(top: 0, leading: 0, bottom: 0, trailing: 16))
```

Comment:
```suggestion
                                            .padding(.trailing, 16)
```
its cleaner to adjust one property than listing bunch of zeros. same goes for every other padding. we use the modifier with EdgeInsets only for special cases where every edge has different padding